### ISSUE TITLE
Decorator @raw_input doesn't work with plugin

### ISSUE THREAD

**Issue by taco-paco** [Shard-Labs/starknet-hardhat-plugin#192]
When calling function with @raw_input decorator getting following error:
`HardhatPluginError: test_raw_input: Expected "calldata_size" to be a felt (Numeric); got: undefined`

**Comment by FabijanC** [comment]
@taco-paco What's the status of this?

**Comment by taco-paco** [comment]
No updates since the last time. Does it become higher priority task?

**Comment by FabijanC** [comment]
No, was just checking. Thanks for the update. Let me know if/when you can get back to this.
